test/development/app-dir/stitching-errors/stitching-errors.test.ts

```diff
@@ -0,0 +1,194 @@
+import { nextTestSetup } from 'e2e-utils'
+import { assertHasRedbox, assertNoRedbox } from 'next-test-utils'
+
+// Remove the location `()` part in every line of stack trace;
+// Remove the leading spaces in every line of stack trace;
+// Remove the trailing spaces in every line of stack trace;
+function normalizeStackTrace(trace: string) {
+  return trace
+    .replace(/\(.*\)/g, '')
+    .replace(/^\s+/gm, '')
+    .trim()
+}
+
+async function getStackFramesContent(browser) {
+  const stackFrameElements = await browser.elementsByCss(
+    '[data-nextjs-call-stack-frame]'
+  )
+  const stackFramesContent = (
+    await Promise.all(stackFrameElements.map((f) => f.innerText()))
+  )
+    .filter(Boolean)
+    .join('\n')
+
+  return normalizeStackTrace(stackFramesContent)
+}
+
+// Remove leading spaces in every line of stack trace
+
+describe('stitching errors', () => {
+  const { next } = nextTestSetup({
+    files: __dirname,
+  })
+
+  it('should log stitched error for browser uncaught errors', async () => {
+    const browser = await next.browser('/browser/uncaught')
+
+    await assertHasRedbox(browser)
+
+    const stackFramesContent = await getStackFramesContent(browser)
+    if (process.env.TURBOPACK) {
+      expect(stackFramesContent).toMatchInlineSnapshot(`
+        "useErrorHook
+        app/browser/uncaught/page.js 
+        Page
+        app/browser/uncaught/page.js"
```

Comment:
I think I saw this somewhere else as well. Why are function name and location on different lines? Does this affect c&p?